src/App.jsx

```diff
@@ -14,57 +16,73 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  const selectGood = good => () => {
+    setSelectedGood(good);
+  };
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
+  const unSelectGood = () => () => {
```

Comment:
```suggestion
  const handleUnSelectGood = () => {
```
- Why is this function wrapped with another function? It doesn't make sense in this case
- Also, in react we developers have a convention on how to name handlers, https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364

Reply:
this is the callback for onClick:
```
const unSelectGood = () => () => { /.../ }
<button onClick={unSelectGood()} />
```
it's the same as
```
const unSelectGood = () => { /.../ }
<button onClick={() => unSelectGood()} />
```
I changed it